labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -71,7 +71,10 @@ object ListManipulationExercise01 {
    * 
    */
   def sortList[T <% Ordered[T]](list: List[T]): List[T] = {
-    error("fix me")
+    if (list.length == 0)
+      return list
+    else
+      return sortList(list.filter((i: T) => i < list.head)) ::: List(list.head) ::: sortList(list.filter((i: T) => i > list.head))
```

Comment:
Oh man, surprise quicksort!

As a further question, how would this quicksort perform on sorted lists?